rholang/src/main/scala/coop/rchain/rholang/Resources.scala

```diff
@@ -55,4 +58,12 @@ object Resources {
     mkTempDir(prefix)
       .flatMap(tmpDir => Resource.make(mkRspace(tmpDir))(_.close()))
   }
+
+
+  def mkRuntime[F[_]: Applicative]( prefix: String,
+                                           storageSize: Long = 1024 * 1024,
+                                           storeType: StoreType = StoreType.LMDB
+                                         )(implicit scheduler: Scheduler): Resource[F, Runtime] =
+    mkTempDir[F](prefix)
+      .flatMap { tmpDir => Resource.pure(Runtime.create(tmpDir, storageSize, storeType))}
```

Comment:
Runtime has `close` method.

Reply:
...but fixing this I shows that Runtime is not consistent in terms of the monad where the acquire/release happen. The Runtime can only run in Task because `close() : Task[Unit]`

This forced me to do some ugly `Task.delay` which I'll address in the next PR